.github/pull_request_template.md

```diff
@@ -19,7 +19,10 @@ _Note_: The title is used as proposal for the maintainer merging the Pull Reques
 
 Explain the **details** for making this change: What existing problem does the Pull Request solve? Why is this feature beneficial?
 
+If you changed the rendering of the BPMN file, add a screenshot.
```

Comment:
I would prefer not mentioning `file` as we are manipulating xml text source for the BPMN diagram.
```suggestion
If your contribution has changed the BPMN rendering, add a screenshot.
```